UnityProject/Assets/Scripts/Objects/Gateway/TransportUtility.cs

```diff
@@ -22,14 +22,12 @@ public class TransportUtility : NetworkBehaviour //Would be a regular static cla
 		/// <param name="transportTo">Destination to transport <paramref name="objectPhysics"/> to.</param>
 		/// <param name="doTileStep">Whether step interactions should trigger on teleport</param>
 		[Server]
-		public static void TransportObject(UniversalObjectPhysics objectPhysics, Vector3 transportTo, bool doTileStep = true, float maintRoomChanceModifier = 0.5f)
+		public static void TransportObject(UniversalObjectPhysics objectPhysics, Vector3 transportTo, bool doTileStep = true, float maintRoomChanceModifier = 0.25f)
 		{
 			if (objectPhysics == null) return; //Don't even bother...
 
 			var dest = transportTo;
-
-			var RandomChance = Random.Range(0, 100000) / 1000f; //100000 / 1000 = 100
-			if (SubSceneManager.Instance.IsMaintRooms && RandomChance <= maintRoomChanceModifier)
+			if (SubSceneManager.Instance.IsMaintRooms && DMMath.Prob(maintRoomChanceModifier))
```

Comment:
Modified to something like
DMMath.Prob(0.001 * maintRoomChanceModifier)

Reply:
done :+1: